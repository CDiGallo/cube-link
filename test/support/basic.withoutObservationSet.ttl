@prefix cube: <https://cube.link/> .
@base <https://example.org/> .

<cube> a cube:Cube .