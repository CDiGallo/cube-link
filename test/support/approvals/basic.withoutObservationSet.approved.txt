_:c14n0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/shacl#ValidationReport> .
_:c14n0 <http://www.w3.org/ns/shacl#conforms> "false"^^<http://www.w3.org/2001/XMLSchema#boolean> .
_:c14n0 <http://www.w3.org/ns/shacl#result> _:c14n1 .
_:c14n1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/ns/shacl#ValidationResult> .
_:c14n1 <http://www.w3.org/ns/shacl#focusNode> <https://example.org/cube> .
_:c14n1 <http://www.w3.org/ns/shacl#resultMessage> "cube:Cube needs at least one cube:ObservationSet" .
_:c14n1 <http://www.w3.org/ns/shacl#resultPath> <https://cube.link/observationSet> .
_:c14n1 <http://www.w3.org/ns/shacl#resultSeverity> <http://www.w3.org/ns/shacl#Violation> .
_:c14n1 <http://www.w3.org/ns/shacl#sourceConstraintComponent> <http://www.w3.org/ns/shacl#MinCountConstraintComponent> .
_:c14n1 <http://www.w3.org/ns/shacl#sourceShape> _:c14n2 .
_:c14n2 <http://www.w3.org/ns/shacl#message> "cube:Cube needs at least one cube:ObservationSet" .
_:c14n2 <http://www.w3.org/ns/shacl#minCount> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:c14n2 <http://www.w3.org/ns/shacl#node> <http://example.org/ObservationSetShape> .
_:c14n2 <http://www.w3.org/ns/shacl#path> <https://cube.link/observationSet> .
